kubernetes/dev/compute-resources.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: beekeeper-api
+spec:
+  replicas: 1
+  template:
+    spec:
+      containers:
+      - name: beekeeper-api
+        resources:
```

Comment:
You can probably reduce the resource requests for these quite a bit. This service probably only needs like 1 or even 0.5 CPU and maybe 10s of MB of memory?

Reply:
Just mentioning that as it may help with your placement of these.

Reply:
It's all up to you guys. I don't care where it lives. I just care that I can easily deploy it on a k8s cluster. I think the honeyhouse config repo is private? The PR does not have anything that needs to be private.

I see these type of overlays as just providing structure an easy way to break down changes so the specific values are not as important to me. The exclude node yaml file will always change based on the status of nodes on Nautilus.